script/os-autoinst-setup-multi-machine

```diff
@@ -45,29 +45,31 @@ install_packages() {
 }
 
 configure_firewall() {
+    echo "Now configuring firewalld..."
     systemctl enable --now firewalld
     default_zone=$(firewall-cmd --get-default-zone)
+    echo "- default_zone=$default_zone"
+    echo "- Creating isotovideo firewall service"
```

Comment:
Is this and other logging meant to be for testing? I would have thought this should be a draft in that case, and not be merged?

Reply:
No in general because it otherwise prints 20x "success" to the console and the user can only guess what that means

Reply:
I guess I wouldn't include the first and last one, but no super strong opinion